src/components/Form.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react"
+import '../styles/_Form.scss'
+const validator = require("email-validator");
+
+const Form = () => {
+  const [email, setEmail] = useState('')
+  const [firstName, setFirstName] = useState('')
+  const [lastName, setLastName] = useState('')
+  const [comments, setComments] = useState('')
+  const [emailError, setEmailError] = useState(false)
+  const [firstNameError, setFirstNameError] = useState(false)
+  const [lastNameError, setLastNameError] = useState(false)
```

Comment:
Just looked at your states and I already can predict what the rest of the code would look like. Love it.